### PR TITLE
[ART-8971] Mirror non-embargoed images to public quay

### PR DESCRIPTION
**Pull request by ashwindasr**
ref. https://issues.redhat.com/browse/ART-8971

### CODE REVIEW

**Comment by ashwindasr** [comment]
/hold

until tested

File: doozer/doozerlib/cli/release_gen_payload.py [hunk 1]
```diff
@@ -761,25 +761,44 @@ async def sync_payloads(self):
...
+            tasks = []
+            for arch, payload_entries in payload_entries_for_each_arch:
                 self.logger.info(f"Building payload files for architecture: {arch}; private: {private_mode}")
-                tasks.append(self.generate_specific_payload_imagestreams(arch, private_mode, payload_entries, multi_specs))
-        await asyncio.gather(*tasks)
+                tasks.append(self.generate_specific_payload_imagestreams(arch,
+                                                                         private_mode,
+                                                                         payload_entries,
+                                                                         multi_specs
+                                                                         ))
```
Comment (sosiouxme) [inline doozer/doozerlib/cli/release_gen_payload.py:800]: when it comes to parameter list indentation, i prefer pretty much any other way that pep8 allows. why? well, not only does this take up more lines and leave your content way out right where you don't see it, but also it needs to be re-indented if you ever refactor names or change something else about the first line.

so i'd suggest:
```suggestion
                tasks.append(self.generate_specific_payload_imagestreams(
                    arch, private_mode, payload_entries, multi_specs))
```

**Review by sosiouxme — COMMENTED** [review]
stylistic suggestions, otherwise:
/lgtm

**Comment by ashwindasr** [comment]
/hold

Until release controller works

**Review by sosiouxme — APPROVED** [review]
/lgtm
[test sync](https://art-jenkins.apps.prod-stable-spoke1-dc-iad2.itup.redhat.com/job/aos-cd-builds/job/build%252Fbuild-sync/336/)

the release-controller doesn't have access to create releases with private content yet but that should come soon enough.

**Pull request merged**